src/index.js

```diff
@@ -579,14 +724,23 @@ const t = defaultProvider();
 export var options = t ? { api: new ParityApi(t) } : null;
 export const bonds = options ? createBonds(options) : null;
 
+// TODO: remove once ParityApi.util.asciiToHex works.
+export function asciiToHex(s) {
```

Comment:
The changes have been merged in - https://github.com/paritytech/js-api/blob/master/util/format.js#L77